core-initrd/latest/debian/rules

```diff
@@ -1,16 +1,60 @@
 #!/usr/bin/make -f
-export DH_VERBOSE=1
 
 include /usr/share/dpkg/default.mk
 
+export DH_VERBOSE=1
+export DH_OPTIONS
+
+BUILDFLAGS:=
+# disable DWARF generation in go linker
+BUILDFLAGS+=-ldflags=-w
+# Disable -buildmode=pie mode on all our 32bit platforms
+# (i386 and armhf). For i386 because of LP: #1711052 and for
+# armhf because of LP: #1822738
+ifeq ($(shell dpkg-architecture -qDEB_HOST_ARCH_BITS),64)
+BUILDFLAGS+=-buildmode=pie
+endif
+
+# check if we need to include the testkeys in the binary
+ifneq (,$(filter testkeys,$(DEB_BUILD_OPTIONS)))
+# if enabled also enable bootloader assets testing and fault injection
+_SNAP_TAGS:=nomanagers,withtestkeys,faultinject
+else
+_SNAP_TAGS:=nomanagers
+endif
+
+ifeq (${FIPSBUILD},1)
```

Comment:
Do we have fips version of pc-kernel?

Reply:
there exists one, but i dunno if it's published

Reply:
We will want it at some point